library/scripts/jjb_compare_xml.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+set -ex
+
+# workaround for old RHELs, we need to install correct tox system-wide
```

Comment:
What about comment at begin of file with name of parameters which is expected by this script.